reference/providers/ecs.md

```diff
@@ -7,13 +7,41 @@ sidebar:
 
 {% include toc %}
 
-## Resource Mapping
+## Resource mapping
 
 ### Server Group
 A Spinnaker **server group** maps to an [Amazon ECS service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html#service_concepts).
 
+There are two ways to specify server group settings for ECS:
+
+* __Inputs (default)__: 
+  Specify the container image, resource limits, and other settings needed to create an [ECS task definition](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definitions.html) and [service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html#service_concepts) directly in the server group. This option supports deploying one container per Amazon ECS task (equivalent to a Spinnaker "instance").
+
+* __Artifact (supported as of 1.15.0)__:
+  Specify a pipeline artifact to use as an ECS Task Definition for the service. The artifact should be a JSON file in the format of an ECS [Register Task Definiton request](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_RegisterTaskDefinition.html). This option also requires that you specify `containerName`:`imageDescription` mappings for each image in your pipeline that you want to be deployed within the service. You can deploy multiple containers (up to 10) per Amazon ECS task using this option.
+
 ### Instance
 A Spinnaker **instance** maps to an Amazon ECS [task](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_run_task.html).  Amazon ECS services manage tasks to ensure desired capacity is reached.
 
 ### Cluster
 An [Amazon ECS cluster](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ECS_clusters.html) does not map to any core Spinnaker concept.  You can choose what cluster you deploy to in your deploy stage parameters.
+
+
+## Operation mapping
+
+### Deploy
+Deploys a new Amazon ECS Service to the specified server group(s). For load balanced services, the previous service will be drained after the new service is considered healthy.
```

Comment:
Should service be upper case or lower case?

Reply:
for consistency, ECS resource types seem to be always lower case; will change.